cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthConfiguration.java

```diff
@@ -395,6 +396,7 @@ public void initializeServletApplicationContext() {
             service.setDescription("OAuth Wrapper Callback Url");
             service.setServiceId(oAuthCallbackUrl);
             service.setEvaluationOrder(Integer.MIN_VALUE);
+            service.setAttributeReleasePolicy(new ReturnAllAttributeReleasePolicy());
```

Comment:
The callback url service should return ALL services as the filtering is done here: https://github.com/apereo/cas/blob/master/cas-server-support-oauth/src/main/java/org/apereo/cas/support/oauth/web/BaseOAuthWrapperController.java#L101 for the real OAuth / OIDC service.